libs/datamodel/core/src/ast/top.rs

```diff
@@ -89,4 +89,9 @@ impl Top {
             _ => None,
         }
     }
+
+    #[track_caller]
+    pub fn unwrap_type_alias(&self) -> &Field {
```

Comment:
Why do we need our own method for this?

Reply:
Purely for convenience when we know we have a `TopId` pointing to an alias, but we could decide to not have these.